resources/CMakeLists.txt

```
@@ -32,6 +32,24 @@ endif()
 if (WITH_SERVER)
   file(GLOB_RECURSE SERVER_RESOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} server/*)
   set(RESOURCES_FILES ${RESOURCES_FILES} ${SERVER_RESOURCE_FILES})
+
+  # Landingpage
+  file(GLOB_RECURSE LANDINGPAGE_SOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} server/src/landingpage/*)
+  set(LANDINGPAGE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/../output/data/resources/server/src/landingpage)
+
+  add_custom_target (yarn-target ALL DEPENDS ${LANDINGPAGE_OUTPUT_PATH}/landingpage.stamp)
```

can we have a more intuitive name for the target?
```suggestion
  add_custom_target (server_landingpage ALL DEPENDS ${LANDINGPAGE_OUTPUT_PATH}/landingpage.stamp)
```
?

---

:+1: sure